alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/resthandler/RestGetMonitorAction.kt

```diff
@@ -62,7 +61,7 @@ class RestGetMonitorAction(settings: Settings, controller: RestController) : Bas
         if (monitorId == null || monitorId.isEmpty()) {
             throw IllegalArgumentException("missing id")
         }
-        val getRequest = GetRequest(SCHEDULED_JOBS_INDEX, SCHEDULED_JOB_TYPE, monitorId)
+        val getRequest = GetRequest(SCHEDULED_JOBS_INDEX, monitorId)
```

Comment:
Expose `if_seq_no` and `if_primary_term` because that is what it will used at the time of update.